website/views/shared/avatar/appearance.jade

```diff
@@ -15,6 +15,9 @@ mixin avatar(opts)
     each klass, item in visualBuffs
       span(ng-if='profile.stats.buffs.#{item}', class='#{klass}')
 
+    // Show flower if using Flower visual buff
+    span(ng-if='profile.stats.buffs.shinySeed', class='hair_flower_{{profile.preferences.hair.flower}}')
```

Comment:
@crookedneighbor @SabreCat What I intended was it would only work with the Flower buff, but I can make it work on all of them